src/test/java/jackbean/command/ParserTest.java

```diff
@@ -0,0 +1,60 @@
+package jackbean.command;
+
+import jackbean.task.TaskList;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ParserTest {
+    TaskList dummyTaskList = new TaskList();
+    @Test
+    public void parseUserInput_invalidCommand_exceptionThrown() {
```

Comment:
I like how the test method names are easy to understand!

Reply:
Thanks bro!